testfiles/avocadoro.postman_collection.json

```diff
@@ -0,0 +1,78 @@
+{
+	"info": {
+		"_postman_id": "977fc1a9-5495-4d37-ace1-f54c24701faa",
+		"name": "avocadoro",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "ping",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{url}}/ping",
+					"host": [
+						"{{url}}"
+					],
+					"path": [
+						"ping"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "accounts - get",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{url}}/accounts",
```

Comment:
wouldn't `account` make more sense than `accounts` in this case

Reply:
In rest api design we use nouns in plural form for endpoints.

https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/